src/views/App.js

```diff
@@ -96,14 +97,20 @@ class App extends Component<AppP> {
         headers: this.getQueryHeaders()
       })
     } else {
+      // includeTableSchema must be passed as a body param when executing an UNSAVED query
       const unsavedQueryURL = `${API_HOST}/v0/${store.queryType}/${
         store.dataset
-      }?includeTableSchema=true`
+      }`
+
+      const bodyParams = {
+        query: store.query,
+        includeTableSchema: true
+      }
 
       res = await fetch(unsavedQueryURL, {
         method: 'POST',
         headers: this.getQueryHeaders(),
-        body: createParams({ query: store.query }).toString()
```

Comment:
is this `createParams` unused in this file now?

Reply:
also did it create an object string before?

Reply:
I'll revert back to using createParams. This was an oversight on my end when troubleshooting against the POST.

Reply:
Actually, scratch that... We do not need to use the createParams function because we are not passing any query params in the POST.

Reply:
i mean it does look like we used to use it to build a POST payload, but your new code should also work just fine